sources/packages/web/src/types/contracts/DataTableContract.ts

```diff
@@ -300,3 +300,54 @@ export const ApplicationWithdrawalUploadHeaders = [
     key: "validations",
   },
 ];
+
+/**
+ * CAS Supplier information header.
+ */
+export const CASSupplierInformationHeaders = [
+  {
+    title: "Date created",
```

Comment:
I am not able to recall the table header latest standards. Should it be `Date created` or `Date Created`? Same for all column headers?

Reply:
Most of the tables have only first in capital.

Reply:
True, but I asked because some content tickets were raised that demanded changing the harder titles casing.